pages/api/gifts/[uuid].ts

```diff
@@ -40,10 +53,15 @@ export default async function handlePrisma(
   }
 }
 
-async function handleGET({ res, queryUUID }: HandlerParams<Gift>) {
+async function handleGET({ res, giftUUID, userData }: HandlerParams<Gift>) {
   const gift = await prisma.gift.findUniqueOrThrow({
     where: {
-      uuid: queryUUID,
+      uuid: giftUUID,
+      AND: {
+        user: {
+          uuid: userData.uuid,
+        },
+      },
```

Comment:
Muistaakseni Prismassa `AND` on oletuksena `where` lausekkeessa, joten kokeilepa toimisiko tämä vähän yksinkertaisempi muoto:
```suggestion
      userUUID: userData.uuid,
```

Reply:
Meikä on painanut hiukan liian kovalla kaasulla, kun tämäkin näköjään on jäänyt korjaamatta :sweat_smile:

Reply:
Pushaa vain niin saadaan korjaus mukaan 👍